.estafette.yaml

```diff
@@ -392,6 +394,18 @@ stages:
         - gke.estafette.internal
         dryrun: true
 
+      test-workloadIdentity:
```

Comment:
Does this test anything that isn't tested in the 2 previous tests with `workloadIdentity: true`? Otherwise you can drop this for the sake of speed and maintainability.